MekHQ/src/mekhq/gui/ForceRenderer.java

```diff
@@ -57,31 +57,29 @@ public Component getTreeCellRendererComponent(JTree tree, Object value, boolean
             }
             String c3network = "";
             StringBuilder transport = new StringBuilder();
-            Unit u = (Unit) value;
-            Person person = u.getCommander();
+            Unit unit = (Unit) value;
+            Person person = unit.getCommander();
             if (person != null) {
                 name = person.getFullTitle();
-                name += " (" + u.getEntity().getCrew().getGunnery() + '/'
-                        + u.getEntity().getCrew().getPiloting() + ')';
-                if (person.needsFixing() || (u.getEntity().getCrew().getHits() > 0)) {
+                name += " (" + unit.getEntity().getCrew().getGunnery() + '/'
+                        + unit.getEntity().getCrew().getPiloting() + ')';
+                if (person.needsFixing() || (unit.getEntity().getCrew().getHits() > 0)) {
                     name = "<font color='" + MekHQ.getMHQOptions().getFontColorNegativeHexColor() + "'>" + name
```

Comment:
I know this isn't your code, but let's get the strings like this replaced with `ReportingUtilities.spanOpeningWithCustomColor(MekHQ.getMHQOptions().getFontColorNegativeHexColor())` and `ReportingUtilities.CLOSING_SPAN_TAG`